style.css

```diff
@@ -0,0 +1,593 @@
+* { 
+    box-sizing: border-box; 
+    padding: 0;
+    margin: 0;
+    font-family: "Chirp";
+    scroll-behavior: smooth;
+    list-style-type: none;
+    text-decoration: none;
+ }
+
+/* STYLES */
```

Comment:
excellent work with the CSS - good use of comments to separate into sections; great use of grids and of advanced CSS styles to achieve overflow and customisation of the page.

Reply:
Thank you. I'm really pleased with the scrolling properties on the webpage